src/components/DivComponent/Div.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+const StyledDiv = styled.div``;
```

Comment:
@Naeem-source instead of this Component, can we utilzie Flex Component ?

Reply:
@yasiriqbal776  yah we can utilize flex compenent ..

Reply:
I remove Div Component and update Merge Request